parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -1119,27 +1120,29 @@ public <D> CloseableIterable<D> build() {
 
         ParquetReadOptions options = optionsBuilder.build();
 
+        NameMapping mapping;
```

Comment:
Optional: You could also check the system property when picking the default value in the constructor or add a method called `nameMapping()` to avoid two variables with similar names in `build()`.

Reply:
I went to add this, but then I thought it was a bit confusing to have both the field and method. I think it should be okay to have an extra variable.